keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -354,7 +357,7 @@ def _call_rpn(self, images, anchors, training=None):
         feature_map = self.backbone(images, training=training)
         feature_map = self.feature_pyramid(feature_map, training=training)
         # [BS, num_anchors, 4], [BS, num_anchors, 1]
-        rpn_boxes, rpn_scores = self.rpn_head(feature_map)
+        rpn_boxes, rpn_scores = self.rpn_head(feature_map, training=training)
```

Comment:
Do we need to add all of these training params?
Seems like we'll end up getting rid of these with the functional subclass?